bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -239,11 +238,21 @@ <h1 class="large">{{_('Check Your Plugins')}}</h1>
     </div>
     <p><a href="#top">{{_('Back to Top')}}</a></p>
 </section>
+
+<!-- dislayed to nonFx users -->
+<div class="not-supported">
+  <header id="main-feature">
+      <h1 class="large">{{_('Check Your Plugins')}}</h1>
+      <p class="preamble">{{_('Keeping your plugins up to date helps Firefox run safely and smoothly.')}}</p>
+  </header>
+
+  <section>
+    <p>{{_('Currently the plugin check service is only available to Firefox users.')}}</p>
```

Comment:
@flodolo so I am just going to wrap it with:

    `{% if l10n_has_tag('plugincheck_nosupport') %}
    ....
    {% endif %}

No else, and then the strings can filter in as they become available so, English will have the string and the button, and all others will just have the button initially.